float64 radius
float64 angular_velocity